lang/lang.txt

```diff
@@ -23523,32 +23523,33 @@ el=Αποκτήστε πρόσβαση σε ρυθμιστές τάσης Intel
 tr=Temel bilgileri ve VR sıcaklığını almak için Intel Seri VID (SVID) Voltaj Regülatörlerine erişin.
 he=גש לרגולטורי מתח Intel Serial VID (SVID) כדי לאחזר מידע בסיסי וטמפרטורת VR.
 
+； Disable this option doesn't affect systems, where the PCH Thermal Device is enabled by BIOS.
```

Comment:
Your change for this section doesn't make sense. Please revert this section back.